warehouse/accounts/models.py

```diff
@@ -293,6 +297,16 @@ def __acl__(self):
                     Permissions.AdminDashboardSidebarRead,
                 ),
             ),
+            (
+                Allow,
+                "group:support",
+                (
+                    Permissions.AdminUsersRead,
+                    Permissions.AdminUsersEmailWrite,
+                    Permissions.AdminUsersAccountRecoveryWrite,
```

Comment:
These two new permissions need to be added to `group:admins` as well, so that admins can also perform these actions.

Reply:
Resolved in bc32652e4af681a0f10332ae962f688337398fdb